### PR TITLE
Hydrocarbon

### PR DESCRIPTION
**Pull request by jake163**

### CODE REVIEW

File: Hydrocarbon-SCF_Guess/compile-hydrocarbons-scf_record.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    scf_guess["Hexane 2Cx3"] = guess_driver("HexaneA.sdf", funct_grps, [1,2,1])
+
+if "HexaneA 3Cx2" not in scf_guess:
+    funct_grps = [[0,12,15,4,2,18,19,14,17,13], [11,3,16,1,6,7,5,8,9,10]]
+    scf_guess["Hexane 3Cx3"] = guess_driver("HexaneA.sdf", funct_grps, [1,1])
+
+with open(filename, 'wb') as infile:
+    pickle.dump(scf_guess, infile)
+
+>>>{'EthaneA 1Cx2': (8, 7), 'PropaneA 1Cx3': (8, 8), 'PropaneA cc-c': (8, 8), 'ButaneA 1Cx4': (10, 10), 'ButaneA 2Cx2': (10, 10), 'ButaneA ccc-c': (10, 10),
```
Comment (jake163) [inline Hydrocarbon-SCF_Guess/compile-hydrocarbons-scf_record.py]: Sorry, I intended to remove that but didn't it seems

**Comment by jake163** [comment]
Use Avogadro for water cluster coordinates?

**Comment by ryanmrichard** [comment]
> Use Avogadro for water cluster coordinates?

Sorry forgot to find you the water cluster link earlier, but it's [here](http://www-wales.ch.cam.ac.uk/~wales/CCD/anant-watcl.html)

**Review by ryanmrichard — APPROVED** [review]

**Comment by jake163** [comment]
> 
> 
> > Use Avogadro for water cluster coordinates?
> 
> Sorry forgot to find you the water cluster link earlier, but it's [here](http://www-wales.ch.cam.ac.uk/~wales/CCD/anant-watcl.html)

Nice. Thank you

**Pull request merged**